routes/api.php

```diff
@@ -13,6 +13,7 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
+Route::prefix('v1')->middleware('client', 'throttle:60,1')->group(function () {
+	Route::post('/add-subscriber', 'SubscriberController@store');
```

Comment:
let's rename to just 
```suggestion
	Route::post('/subscriber', 'SubscriberController@store');
```